war/src/main/java/io/lumeer/engine/controller/CollectionMetadataFacade.java

```diff
@@ -677,7 +682,11 @@ public void dropAttributeConstraint(String collectionName, String attributeName,
     * @return true if user can read the collection
     */
    public boolean checkCollectionForRead(String collectionName, String user) {
-      return securityFacade.checkForRead(getCollectionMetadataDocument(collectionName), user);
+      try {
+         return securityFacade.checkForRead(getCollectionMetadataDocument(collectionName), user);
+      } catch (CollectionMetadataDocumentNotFoundException e) {
+         return true; // if metadata is not found, we allow access
```

Comment:
I thought it would be better to check existence of the document, but if it does not exist, something bad is happening which should never happen.